Package.swift

```diff
@@ -0,0 +1,9 @@
+import PackageDescription
+
+let package = Package(
+    name: "GRDB.swift",
+    dependencies: [
+        .Package(url: "https://github.com/groue/SQLiteSDK.git", majorVersion: 0)
+    ],
+    exclude: ["GRDB.xcworkspace", "Playgrounds", "SQLCipher", "SQLiteCustom", "Support", "DemoApps", "Tests", "Documentation"]
```

Comment:
OK, thanks for this detailed info. I don't feel like holding this PR just because we can't decide how to layout the files and directories: the status quo is good enough, even if we have to maintain an exclude list.